terroroftinytown/tracker/model.py

```diff
@@ -436,22 +436,30 @@ def has_results(cls):
             return bool(result)
 
     @classmethod
-    def get_count(cls):
+    def get_count(cls, project_id=None, **unused_kwargs):
         with new_session() as session:
-            min_id = session.query(func.min(Result.id)).scalar() or 0
-            max_id = session.query(func.max(Result.id)).scalar() or 0
+            q = session.query(func.count(Result.id))
+            if project_id is not None and project_id != 'None':
+                q = q.filter(Result.project_id == project_id)
+            return q.scalar()
+
 
-            return max_id - min_id
```

Comment:
`unused_kwargs` allows it be called the same way as `get_results` is called (i.e. `**args`). It could be changed to explicitly only be passed `project_id` if you'd prefer.

Regarding `count()` -- I did see that your way was faster (when not providing `project_id`). I thought the difference in performance wasn't worth it, as I missed the fact that it is checked every time an item is checked out. I'll add it back in.